test-clients/src/main/java/com/hedera/services/legacy/smartcontract/V3V4Before.java

```diff
@@ -135,27 +135,10 @@ private void loadGenesisAndNodeAcccounts() throws Exception {
 
   }
 
-  private Transaction createQueryHeaderTransfer(AccountID payer, long transferAmt)
-      throws Exception {
-    Timestamp timestamp = RequestBuilder
-        .getTimestamp(Instant.now(Clock.systemUTC()).minusSeconds(13));
-    Duration transactionDuration = RequestBuilder.getDuration(30);
-
-    //KeyPair pair = new KeyPairGenerator().generateKeyPair();
-    //byte[] pubKeyBytes = ((EdDSAPublicKey) pair.getPublic()).getAbyte();
-    //String pubKey = HexUtils.bytes2Hex(pubKeyBytes);
-    //Key key = Key.newBuilder().setEd25519(ByteString.copyFrom(pubKey.getBytes())).build(); // used later
-    SignatureList sigList = SignatureList.getDefaultInstance();
-		 /* Transaction transferTx = RequestBuilder.getCryptoTransferRequest(
-					 payer.getAccountNum(), payer.getRealmNum(), payer.getShardNum(), nodeAccount.getAccountNum(), nodeAccount.getRealmNum(), nodeAccount.getShardNum(),
-					 50, timestamp, transactionDuration, false, "test", sigList,
-					 payer.getAccountNum(), -100l, nodeAccount.getAccountNum(), 100l);*/
-
-    Transaction transferTx = TestHelper.createTransfer(payer, accountKeys.get(payer).get(0),
+  private Transaction createQueryHeaderTransfer(AccountID payer, long transferAmt) {
+    return TestHelper.createTransfer(payer, accountKeys.get(payer).get(0),
         nodeAccount, payer,
         accountKeys.get(payer).get(0), nodeAccount, transferAmt);
```

Comment:
Are we creating same transfer transaction as original code? For example, the duration of transaction in new code is missing.

Reply:
I just removed the unused code. I you look at the old code you can see that we are not using the transaction duration and the siglist.

Reply:
Got it.